release-notes/unreleased/IGAPP-1023-footer-overlay-for-map-view-desktop.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-1023
+show_in_stores: true
+platforms:
+  - web
+en: Footer is now display as an overlay within the map.
+de: Der Footer wird nun als Overlay innerhalb der Karte angezeigt.
```

Comment:
JFYI: The german release note is never shown/used for web only or not shown in the store release notes.

Reply:
alright thx for info. removed